debug/stanza.proj

```diff
@@ -0,0 +1,16 @@
+packages stz-debug/* defined-in "."
+
+package stz-debug/debugger requires :
+  ccfiles: "debugger.c"
```

Comment:
Moved these requirements to a `requires` statement instead of in the build target itself.